src/components/SelectedTapMobile/SelectedTapDetails.js

```diff
@@ -235,4 +226,36 @@ function SelectedTapDetails(props) {
   );
 }
 
+/**
+ * Get a list of tags to display for a resources
+ * @param {ResourceEntry} resource
+ * @returns {Array<string>} A list of tags to display in the UI
+ */
+function getTagsFromResource(resource) {
+  // First, get the tags
+  const tags = [
+    resource.water,
+    resource.food,
+    resource.forage,
+    resource.bathroom
+  ]
+    .filter(Boolean) // Filter out any missing resources if it is not that type
+    .flatMap(item => item.tags);
+
+  // Then , get resource-specific information
+  if (resource.water) {
+    tags.push(...(resource.water.dispenser_type || []));
+  }
+  if (resource.food) {
+    tags.push(...(resource.food.food_type || []));
+    tags.push(...(resource.food.distribution_type || []));
+    tags.push(...(resource.food.organization_type || []));
+  }
+  if (resource.forage) {
+    tags.push(resource.forage.forage_type);
+  }
```

Comment:
Just to confirm, do bathroom resources not have any resource-specific information? Want to make sure that wasn't an unintended omission

Reply:
Confirmed that bathrooms do not have resource-specific information